example/imagery/plugin.js

```diff
@@ -168,7 +168,7 @@ function getImageUrlListFromConfig(configuration) {
 }
 
 function getImageLoadDelay(domainObject) {
-    const imageLoadDelay = domainObject.configuration.imageLoadDelayInMilliSeconds;
+    const imageLoadDelay = parseInt(domainObject.configuration.imageLoadDelayInMilliSeconds, 10);
```

Comment:
Nice catch! One issue I noticed with using `parseInt()` is that e-notation values such as `2e3` don't get converted correctly since parsing stops at the first non-numeric character. We can use `Number()` to convert the type, and then force an integer value using `Math.trunc()`, since we probably don't want fractional millisecond values:


```suggestion
    const imageLoadDelay = Math.trunc(Number(domainObject.configuration.imageLoadDelayInMilliSeconds));
```

Reply:
I've implemented the suggested fix and re-ran the E2E test; it all seems to work.  How is it working for you?